src/EnergyPlus/MixedAir.cc

```diff
@@ -1471,15 +1466,9 @@ void GetOAControllerInputs(EnergyPlusData &state)
                     if (ZoneListNum > 0) {
                         for (int ScanZoneListNum = 1; ScanZoneListNum <= state.dataHeatBal->ZoneList(ZoneListNum).NumOfZones; ++ScanZoneListNum) {
                             // check to make sure zone name is unique (not listed more than once)...
-                            int ZoneNum = state.dataHeatBal->ZoneList(ZoneListNum).Zone(ScanZoneListNum);
```

Comment:
So, I mistakenly deleted this line. How did it build, then? Is there another `ZoneNum` that's still in scope here?

Reply:
Wise choice to change the var name. Same name in the outer loop.